modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_sidebar/CriteriaSidebarSearchBar.es.js

```diff
@@ -55,6 +55,11 @@ class CriteriaSidebarSearchBar extends Component {
 							onClick={
 								searchValue ? this._handleClear : undefined
 							}
+							title={
```

Comment:
Ei @beltranrengifo , what do you think of using aria-label in this case so that no tooltip is displayed when we hover the button. The "Search" is already present in the input. It's just a suggestion.

Reply:
I like tooltips, but yes: in normal state it's redundant. However, when the input is dirty it shows `clear` which can be helpful. Maybe we can ask for help to UI team